test/mkcheck.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
OK.  I actually thought about that.  I will add it although I'm not really sure that we are copyrighting this code.